api2-war/src/main/java/eu/europeana/api2/v2/web/controller/ContentReuseFrameworkController.java

```diff
@@ -29,7 +53,12 @@
     @Resource
     private SearchService searchService;
 
-  
+    @Resource(name = "corelib_web_contentReuseFrameworkService")
```

Comment:
Is qualifier needed? Is only required when there are more beans defined with the same class type. Otherwise Spring with automaticly use the class type to match and a name is not needed here.